### DISCUSSION TITLE
Why beta?

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#205] [General]
The [Quick Start](https://github.com/framesurge/perseus) and [Cargo.toml.example](https://github.com/framesurge/perseus/blob/main/examples/comprehensive/tiny/Cargo.toml.example) tell us to use 0.4.0-beta.10, seems a beta version, is that stable enough?

**Comment by lukechu10** [comment]
That's because you are looking at the examples built for the latest version of perseus which is `0.4.0-beta.10`. The examples are built with the version of perseus on the `main` branch.

The latest beta version should be pretty stable. The reason why `0.4.0` hasn't been released yet is mostly because they are a few more features that are not yet implemented.

**Reply by ghost** [reply]
[This page](https://framesurge.sh/perseus/en-US/docs/0.4.x/intro/) says the beta isn't stable yet.

**Reply by arctic-hen7** [reply]
The difference between beta and stable is mostly a technicality. As stated, there are still a few features that I'd like to implement before v0.4.x goes 'stable'. The reason it's released as a beta is because I want to be able to release new features rapidly and have them tested in the real world, rather than creating a pile-up of features that people have to wait ages for.

I would recommend using v0.4.x at this stage, but just be aware that there is still a bit more work to be done, and new beta versions may well have a few bugs. If you're concerned, I would just pin to `=0.4.0-beta.10`, which the official website is currently running on. Please report any bugs you find, though!
